packages/adapter-squid/.env.example

```diff
@@ -0,0 +1,3 @@
+# Only used to override global settings for testing
+primaryPrivateKey=0x1234456789abcdef123456789abcdef123456789abcdef123456789abcdef
+primaryAddress=0x1234456789abcdef123456789abcdef123456789abcdef123456789abcdef
```

Comment:
same idea as in other `.env.example`, when no working defaults can be provided, it could be better to let it empty to allow the code to explicitly tell devs to set these values (rather then thinking they're set when in fact the execution will fail later on because the private key is set, but invalid)

```suggestion
# Only used to override global settings for testing
# i.e. 0x1234456789abcdef123456789abcdef123456789abcdef123456789abcdef
primaryPrivateKey=
# i.e. 0x1234456789abcdef123456789abcdef123456789abcdef123456789abcdef
primaryAddress=
```